app-config.yaml

```diff
@@ -1,6 +1,8 @@
 app:
   title: Backstage Example App
   baseUrl: http://localhost:3000
+  support:
+    url: https://github.com/backstage/backstage/issues
```

Comment:
code supports both a URL and a Title; might we want to add a sample here?

brings up other points for me, so in total:

* should title example be set here? (even if it's optional in code)
* should this title and url actually be here but commented out; i.e., note they default, but CAN be customized if you want to?
* Should whatever you end up with from the above _also_ be added to the template in the scaffolded app config for create-app?

Reply:
The recent updates include a full example of the new usage in the root `app-config.yaml`.